theme/islandora-videojs.tpl.php

```diff
@@ -1,15 +1,23 @@
-<video id="islandora_videojs" class="video-js vjs-default-skin" controls
- preload="auto" width="640" height="264" 
+<video id="islandora_videojs" <?php if (empty($responsive)): ?>width="640" height="360"<?php endif; ?>
+  <?php if (empty($center_play_button)): ?>class="video-js vjs-default-skin"
+  <?php else: ?>class="video-js vjs-default-skin vjs-big-play-centered" <?php endif; ?>
+    controls
+  preload="auto"
   <?php if (isset($tn)): ?>
     poster="<?php print $tn; ?>"
   <?php endif; ?>
- data-setup="{}">
+  <?php if ($responsive != 0): ?>
```

Comment:
Also just noticed you are doing a `if(empty($responsive))` above and here a `if ($responsive != 0)`. Maybe you want return a boolean when you set this variable in `video_js.module` then it could just be `if ($responsive)`?

Something like
```
$variables['responsive'] = (bool) variable_get('islandora_videojs_responsive', FALSE);
```